VeganVirus/VeganVirus/VeganVirus.vcxproj

```diff
@@ -140,11 +140,14 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClInclude Include="Action.h" />
+    <ClInclude Include="Animation.h" />
```

Comment:
why do you add space like this "Action.h  /> and not do like this "Action.h />? consider fixing it.

Reply:
This file is automatically changed by visual studio, and we'll only change it manually to fix merge conflicts.